### ISSUE TITLE
[FEATURE REQ] Fetching Bookmarks

### ISSUE THREAD

**Issue by sujanchegu** [pesos/browser-history#51]
**Is your feature request related to a problem? Please describe.**
Since the data related to the bookmarks will also be stored in sql files, adding support to fetch those would be an enhancement and additional functionality.

**Describe the solution you'd like**
In the `Outputs` object, add a member function similar to `get` to fetch the bookmarks of the browsers associated with that `Outputs` object.

Please comment on this if you have any additional thoughts or ideas.

**Comment by Samyak2** [comment]
This will have to be added to each browser. It will be implemented in `generic.Browser` and the browser classes will have to provide a SQL for getting the bookmarks.

**Comment by sujanchegu** [comment]
Having a separate branch for bookmarks would be ideal instead of adding it to master itself.

**Comment by Samyak2** [comment]
Yes, better to have it in a separate branch for now

**Comment by lucasace** [comment]
Heyy, I think I can do this, 
1. Can I get assigned
2. Whats the output you looking for? , Is date added and url enough?

**Comment by sujanchegu** [comment]
Yes. I actually don't know how the bookmarks are actually stored but is it possible to fetch the folder that it is stored along with it? Something like this:
![image](https://user-images.githubusercontent.com/43275517/95422302-af712080-095c-11eb-9409-1041e5d12332.png)

You can see that some bookmarks are stored in folders while some are just put to bookmarks bar. I know this is a troublesome task and might even be not useful, but if you feel this would be easy to implement, then try it. Otherwise no need.

**Comment by lucasace** [comment]
Yea sure ill give it a try

**Comment by lucasace** [comment]
I opened a Draft Pr, could you check it?

**Comment by lucasace** [comment]
I think you can check everything except Safari, UC,IE and Epic

**Comment by Samyak2** [comment]
> I think you can check everything except Safari, UC,IE and Epic

Done

> Also Safari doesn't store its bookmarks in a SQL format, it stores it in a plist format
> Location : Library/Safari/Bookmarks.plist

Cool. Looks like python has a standard library to deal with `plist` files called [plistlib](https://docs.python.org/3/library/plistlib.html) :fire:
